app-ios/Sources/AnnouncementFeature/AnnouncementView.swift

```diff
@@ -0,0 +1,83 @@
+import appioscombined
+import ComposableArchitecture
+import Model
+import SwiftUI
+
+public struct AnnouncementState: Equatable {
+    public var announcements: [AnnouncementsByDate] = []
+
+    public init() {}
+}
+public enum AnnouncementAction {
+    case load
+    case fetched([AnnouncementsByDate])
+}
+
+public struct AnnouncementEnvironment {
+    public let repository: AnnouncementsRepository
+
+    public init(repository: AnnouncementsRepository) {
+        // TODO: Replace FakeRepository with DataRepository
+        self.repository = FakeAnnouncementsRepository()
```

Comment:
Apparently, I couldn't fetch announcements from server so, instead, I used FakeRepository.
@ry-itto Could you check it?🙏

Reply:
Currently, maybe server returns empty announcements  and Android app fetches empty list as well. So I think it's okey!

But I'll ask server team tomorrow, please wait for it 🙏🏼

Reply:
I asked server team, they said current state is correct!
So please change here with using initializer variable 🙏🏼